src/particles/plasma/PlasmaParticleContainer.H

```diff
@@ -137,6 +138,18 @@ public:
     /** Returns name of the plasma */
     const std::string& GetName () const {return m_name;}
 
+    /** Compute reduced plasma diagnostics of current slice, store in member variable
```

Comment:
Let's try to stick to the 1 name in-situ, otherwise we could have confusion with other diagnostics.
```suggestion
    /** Compute in-situ plasma diagnostics of current slice, store in member variable
```